src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.CompilationData.cs

```diff
@@ -102,54 +119,8 @@ internal void ClearDeclarationAnalysisData(SyntaxReference declaration)
                     }
 
                     _declarationAnalysisDataMap.Remove(declaration);
```

Comment:
I think now you can just do 
_declarationAnalysisDataMap.Remove(declaration);

without TryGetValue since you don't need data anymore

Reply:
Yes, sure.